README.md

```diff
@@ -214,6 +214,33 @@ As administrator
 pip install --user robotframework-autoitlibrary
 ```
 
+General usage
+====
+
+The framework comes with its own CLI utility exposed through the make command. To enlist all commands available run:
+
+```
+make
+```
+
+Settings / Config
+====
+
+The framework has 3 files that serve its settings.
+
+1. settings.robot (Default distributed settings that ship as standard).
+2. settings-override.py (A templated custom override file that is not checked in, and can be modified).
+3. secrets.robot (A templated file that is not checked in, and holds all sensitive information such as passwords).
+
```

Comment:
I think we need to highlight somewhere that variables are set in a different way in the Python file from the Robot files, possibly with examples such as:

```
${BASE_URL}                         https://ccmsebs.uat.legalservices.gov.uk/OA_HTML/AppsLocalLogin.jsp
```
```
BASE_URL = 'https://ccmsebs.uat.legalservices.gov.uk/OA_HTML/AppsLocalLogin.jsp'
```